libraries/AP_NavEKF2/AP_NavEKF2_PosVelFusion.cpp

```diff
@@ -46,6 +46,16 @@ void NavEKF2_core::ResetVelocity(void)
             // clear the timeout flags and counters
             velTimeout = false;
             lastVelPassTime_ms = imuSampleTime_ms;
+        } else if (imuSampleTime_ms - extNavVelMeasTime_ms < 250) {
+            // use external nav data as the 2nd preference
+            // correct for antenna position
```

Comment:
This is not an antenna, replace with "external navigation sensor"

Reply:
thank you